pkg/storj/bucket.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storj
+
+import (
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+)
+
+var (
+	// ErrBucket is an error class for general bucket errors
+	ErrBucket = errs.Class("bucket")
+
+	// ErrNoBucket is an error class for using empty bucket name
+	ErrNoBucket = errs.Class("no bucket specified")
+
+	// ErrBucketNotFound is an error class for non-existing bucket
+	ErrBucketNotFound = errs.Class("bucket not found")
+)
+
+// Bucket contains information about a specific bucket
+type Bucket struct {
+	ID                   uuid.UUID
+	Name                 string
+	ProjectID            uuid.UUID
+	Attribution          string
+	Created              time.Time
+	PathCipher           CipherSuite
+	SegmentsSize         int64
```

Comment:
These are defaults not the sizes.

Reply:
I feel like this Bucket struct is used in other places where these are the actual sizes and not the defaults.  Or I guess what does "defaults" mean?  What is set by default in the code? Or what value gets passed into the uplink config? Or what?

Reply:
buckets in general doesn't' have a size or redundancy scheme. these values are stored as defaults for objects uploads.